### PR TITLE
push ESIP first

### PR DESCRIPTION
**Pull request by ocefpaf**
The pangeo-notebook fails on/off to solve for the environment but the pangeo-esip does not. Let's deploy the latter first so we can help @rsignell-usgs get an image for his demo in January.

PS: I'm also checking why the pangeo-notebook is failing to solve in an acceptable amount of time. That is a conda problem regardless of the CI used here.

### CODE REVIEW

**Comment by ocefpaf** [comment]
@rsignell-usgs the `pangeo-notebook` failed and the `pangeo-esip` passed, as expected. Locally I can get the `pangeo-notebook` env solution down to a reasonable amount of time with the `strict` channel option (which is the only safe way to use conda-forge).

Unfortunately that change must happen in `repo2docker` and this proposal was rejected in the past. See https://github.com/jupyter/repo2docker/pull/714

I disagree with the reasons to avoid `strict` there and would argue for a config option at least, but we'll probably have better luck changing the way conda handle `environment.yaml` files (that is something under development at the moment).

TL;DR hopefully with the changes here we'll get at least the ESIP image to upload.

**Comment by rsignell-usgs** [comment]
Thanks for your time here @ocefpaf, much appreciated.   Merging to get the ESIP container built.
